examples/horovod/entrypoint.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
```

Comment:
I am wondering if it is needed for every MPIJob. Should we add some comments about why to introduce such a entrypoint script?

Reply:
Yes, something like this will always be needed.
The problem is that SSH is very picky about not just file permissions, but also the permissions of the directory. Unfortunately, kubernetes doesn't allow to change the permissions of the mount folder.

I'll update the README once I'm done with all development (tests included). The idea is: the controller will mount the keys in /mnt/ssh. It's up to the container image to set those files in the appropriate location for their user.

Reply:
FTR, I tried to use mount the files in /root/.ssh or used them from /mnt/ssh by setting `AuthorizedKeysFile` and `IdentityFile` in the ssh configuration. However, SSH complained about the folder permissions :(

Reply:
Hold on, I'm investigating if I can do the same in an init container.

Reply:
I'm surprised, but it worked. Now, I'm not convinced that it will work if the containers don't run as root, but I'll check that later. I added a bullet point in #373.